src/styles/blocks/catalog.scss

```diff
@@ -18,6 +18,13 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition-duration: $transition-duration;
```

Comment:
You also need to specify the properties you change on hover in transition rule.

Example:

`transition: color $transition-duration, transform $transition-duration;`